NewHorizons/NewHorizons.csproj

```diff
@@ -27,12 +28,32 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  <ItemGroup>
-    <Content Include="NewHorizons.csproj.user" />
-  </ItemGroup>
   <ItemGroup>
     <Reference Include="PacificEngine.OW_CommonResources">
       <HintPath>$(OuterWildsModsDirectory)\PacificEngine.OW_CommonResources\PacificEngine.OW_CommonResources.dll</HintPath>
     </Reference>
   </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```

Comment:
This will auto build
- A test branch into the mod load
- A deployment zip file

Reply:
This does require powershell to run.